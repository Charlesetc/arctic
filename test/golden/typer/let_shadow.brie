define main [ : x [
  x a
  define x 3
  define y "hi"
  x
] ]
