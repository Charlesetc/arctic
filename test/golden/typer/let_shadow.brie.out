{[3, 4]=>Literal(subtype=string), [6, 0]=>Open_function, [1, 2, 5, 7]=>Literal(subtype=integer), [8, 9]=>Function_literal(takes=0, returns=1)}